### PR TITLE
Newgetstats

### PR DESCRIPTION
**Pull request by timcadman**
> prerequisites:
> - install.packages("devtools")
> - install.packages("styler")
> - install.packages("pkgdown")

#### In general:

* [ ] Did you test the code against `https://armadillo.test.molgenis.org`?
* [ ] Have you ran `styler:::style_active_pkg()` to style the code-files
* [ ] Have you ran `devtools::check()` to be sure documentation is rerendered and code is quality checked?

#### Vignettes:

1. [ ] Did you create an .orig file and add it to the pre-render.R file?
2. [ ] Did you run the the pre-render.R file to regenerate the vignettes
3. [ ] Did you run `pkgdown::build_site()` to preview the documentation

### CODE REVIEW

**Comment by timcadman** [comment]
Hmm not sure how all of these other commits have ended up on this branch!! Still trying to get to grips with git :D

**Comment by timcadman** [comment]
Ah ok, now I've merged the other branch its caught up :)

File: R/lm-tab.R [hunk 1]
```diff
@@ -35,17 +35,30 @@ dh.lmTab <- function(model = NULL, type = NULL, coh_names = NULL,
...
   direction <- arg_match(direction)
   ci_format <- arg_match(ci_format, c("paste", "separate"))
+  family <- arg_match(family, c("gaussian", "binomial"))
 
   if (direction == "long" & ci_format == "paste") {
     warning("It is not possible to paste CIs in long format. Argument ignored")
   }
 
+  if(family == "gaussian"){
+
```
Comment (sidohaakma) [inline R/lm-tab.R:45]: Please run the styler ;-D.

**Review by sidohaakma — CHANGES_REQUESTED** [review]
Run the styler and the check function!

File: R/lm-tab.R (hunk 1 above)
Comment (sidohaakma) [reply R/lm-tab.R:45]: styler::style_package(). You can ignore the check function then

**Comment by timcadman** [comment]
Ok I think ive fixed it all now - can you review and approve?

**Review by sidohaakma — APPROVED** [review]

**Pull request merged**